src/axom/inlet/Table.hpp

```diff
@@ -921,9 +995,129 @@ class Table : public Verifiable
    * \param [in] name The name of the array object in the input file
    *****************************************************************************
    */
-  std::vector<std::pair<std::string, std::string>> arrayIndicesWithPaths(
+  std::vector<std::pair<std::string, std::string>> containerIndicesWithPaths(
     const std::string& name) const;
 
+  /*!
+   *****************************************************************************
+   * \brief Get an array represented as an unordered map from the input file
+   * of primitive type
+   *
+   * \param [out] map Unordered map to be populated with array contents
+   *
+   * \return Whether or not the array was found
+   *****************************************************************************
+   */
+  template <typename T>
+  typename std::enable_if<detail::is_inlet_primitive<T>::value, bool>::type
+  getArray(std::unordered_map<int, T>& map) const
+  {
+    map.clear();
+    if(!axom::utilities::string::endsWith(m_name, CONTAINER_GROUP_NAME))
+    {
+      return false;
+    }
+    for(auto& item : m_fieldChildren)
+    {
+      auto pos = item.first.find_last_of("/");
```

Comment:
Minor -- It might also be a good idea to have a utility function to find this index.